src/admin/python/lsst/qserv/admin/cli/entrypoint.py

```diff
@@ -371,20 +577,27 @@ def worker_xrootd(
     help=f"{repl_connection_option.keywords['help']} {socket_option_help}"
 )
 @debug_option()
+@cmd_option(default="qserv-replica-worker --qserv-worker-db={{db_admin_uri}} {% for arg in extended_args %}{{arg}}  {% endfor %}")
```

Comment:
@iagaponenko I don't see a comment re. `db_admin_url`, is it missing or can you point me to a file & function?

`entrypoint` does not do anything with the `--config` option right now. What are you imagining it might need to do, or how would it need to be part of the `entrypoint` template processing?

Reply:
It's in the same line for which I made the comment (look for `--qserv-worker-db`):
```
@cmd_option(default="qserv-replica-worker --qserv-worker-db={{db_admin_uri}} ...
```